tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -0,0 +1,426 @@
+
+            # Make one of the node down where the prometheus pod is hosted
+            nodes.restart_nodes([pod_node_obj])
+
+            # Validate all nodes are in READY state
+            wait_for_nodes_status()
+
+        # Check the node are Ready state and check cluster is health ok
+        self.sanity_helpers.health_check()
+
+        # Check all the prometheus pods are up
+        for pod_obj in pod_obj_list:
+            wait_for_resource_state(resource=pod_obj, state=constants.STATUS_RUNNING)
+
+        # Check for the created pvc metrics after restarting node where prometheus pod is hosted
+        for pod_obj in pods:
+            assert check_pvcdata_collected_on_prometheus(pod_obj.pvc.name), (
+                f"On prometheus pod for created pvc {pod_obj.pvc.name} related data is not collected"
+            )
+            log.info(f"On prometheus pod for created pvc {pod_obj.pvc.name} related data is collected")
+
+    @pytest.mark.polarion_id("OCS-709")
+    def test_monitoring_after_rebooting_master_node(self, nodes, pods):
+        """
+        Test case to validate rebooting master node shouldn't delete
+        the data collected on prometheus pod
+
+        """
+
+        # Get the master node list
+        master_nodes = get_typed_nodes(node_type='master')
+
+        # Reboot one after one master nodes
+        for node in master_nodes:
+            nodes.restart_nodes([node])
+
+            wait_for_nodes_status_and_prometheus_health_check(pods)
+
+        # Check the node are Ready state and check cluster is health ok
+        self.sanity_helpers.health_check()
+
+    @pytest.mark.polarion_id("OCS-710")
+    def test_monitoring_after_rebooting_node_where_mgr_is_running(self, nodes, pods):
+        """
+        Test case to validate rebooting a node where mgr is running
+        should not delete the data collected on prometheus pod
+
+        """
+
+        # Get the mgr pod obj
+        mgr_pod_obj = pod.get_mgr_pods()
+
+        # Get the node where the mgr pod is hosted
+        mgr_node_obj = pod.get_pod_node(mgr_pod_obj[0])
+
+        # Reboot the node where the mgr pod is hosted
+        nodes.restart_nodes([mgr_node_obj])
+
+        # Validate all nodes are in READY state
+        wait_for_nodes_status()
+
+        # Check the node are Ready state and check cluster is health ok
+        self.sanity_helpers.health_check()
+
+        # Check for ceph health check metrics is updated with new mgr pod
+        wait_to_update_mgrpod_info_prometheus_pod()
+
+        # Check for the created pvc metrics after rebooting the node where mgr pod was running
+        for pod_obj in pods:
+            assert check_pvcdata_collected_on_prometheus(pod_obj.pvc.name), (
+                f"On prometheus pod for created pvc {pod_obj.pvc.name} related data is not collected"
+            )
+
+    @pytest.mark.polarion_id("OCS-711")
+    def test_monitoring_shutdown_and_recovery_prometheus_node(self, nodes, pods):
```

Comment:
This looks similar to `test_monitoring_when_one_of_the_prometheus_node_down`. For shutdown and recovery, let's keep the node down for `x` time instead of rebooting node.

Reply:
Stop the node and waiting for 20 seconds now